gix-traverse/src/commit/mod.rs

```diff
@@ -0,0 +1,56 @@
+use gix_object::FindExt;
+use smallvec::SmallVec;
+
+/// Simple ancestors traversal
+pub mod ancestors;
+pub use ancestors::{Ancestors, Sorting};
```

Comment:
I think I will refactor the way `ancestors` is made to be more aligned with what's in `topo`, which would also avoid having to `pub use ` anything here for historical reasons.

There is other issues I have with `ancestors` mostly around unnecessary complexity that I'd like to resolve once and for all then.